egs/wsj/asr2/run.sh

```diff
@@ -318,8 +318,8 @@ fi
 
 if [ ${stage} -le 5 ] && [ ${stop_stage} -ge 5 ]; then
     echo "stage 5: Decoding"
-    nj=32
-    if [ ${n_average} -gt 1 ]; then
+    nj=${n_decode_job}
+    if [ ${n_average} -gt 1 ] && [ ! -e ${expdir}/results/model.last${n_average}.avg.best ]; then
```

Comment:
I am wondering about this part.
cause if the `avg.best` exists, then `recog_model` var will not change. 
If that is the case isn't better to put a second if inside this part and before `average_checkpoints.py`?

Reply:
Thanks. I think it is better to remove this line from here. I just added this `[ ! -e ${expdir}/results/model.last${n_average}.avg.best ]` when I test consistency of decoding on refactoring.